### PR TITLE
Moved chara from support to moderator page, added new mods

### PR DESCRIPTION
**Pull request by ItsCharaHere**

### CODE REVIEW

File: wiki/about/moderators.md [hunk 1]
```diff
@@ -92,3 +92,22 @@ Any stroopwafels?
 **Roles:** Moderator  
 
 [Twitch](https://twitch.tv/olafstad)
+
+## Chara_
+gaming 101
+
+**Roles:** Support, 3D Artists, Moderator
+
+[Discord](https://discord.gg/NXnPYEh) | [Youtube](https://www.youtube.com/c/CharaHere)
```
Comment (Caeden117) [inline wiki/about/moderators.md:101]: Are links to discords allowed?

Comment (ItsCharaHere) [reply wiki/about/moderators.md:101]: I mean, the discord is relevant to beat saber

Comment (megalon) [reply wiki/about/moderators.md:101]: I don't see why not

Comment (Caeden117) [reply wiki/about/moderators.md:101]: I was just wondering. Thanks!

**Pull request merged**